locales/sv.lua

```diff
@@ -1,19 +1,19 @@
 local Translations = {
     notify = {
-        ["hud_settings_loaded"] = "HUD-inställningar laddade!",
-        ["hud_restart"] = "HUD startas om!",
+        ["hud_settings_loaded"] = "HUD Inställningar laddade!",
+        ["hud_restart"] = "HUD startar om!",
         ["hud_start"] = "HUD är startad!",
-        ["hud_command_info"] = "Detta kommando nollställer tidigare HUD-inställningar!",
-        ["load_square_map"] = "Fyrkantid överblickskarta laddas...",
-        ["loaded_square_map"] = "Fyrkantig överblickskarta har laddats!",
-        ["load_circle_map"] = "Cirkulär överblickskarta laddas...",
-        ["loaded_circle_map"] = "Cirkulär överblickskarta har laddats!",
-        ["cinematic_on"] = "Bioläge - På!",
-        ["cinematic_off"] = "Bioläg - Av!",
-        ["engine_on"] = "Motor startad!",
-        ["engine_off"] = "Motor stängs av!",
-        ["low_fuel"] = "Bränslenivå - låg!",
-        ["access_denied"] = "Du har inte rätt behörighet!",
+        ["hud_command_info"] = "Detta kommando nollställer dina nuvarande HUD-inställningar!",
+        ["load_square_map"] = "Kvadratisk karta laddas!",
+        ["loaded_square_map"] = "Kvadratisk karta har laddats!",
+        ["load_circle_map"] = "Cirkelkartan laddas!",
+        ["loaded_circle_map"] = "Cirkelkartan har laddats!",
+        ["cinematic_on"] = "Bioläge på!",
+        ["cinematic_off"] = "Bioläge av!",
+        ["engine_on"] = "Motorn startadar!",
+        ["engine_off"] = "Motorn stängs av!",
+        ["low_fuel"] = "Låg bränslenivå!",
+        ["access_denied"] = "Du är inte behörighet att göra detta!",
```

Comment:
Supposed to be "Du _har_ inte _behörighet_ att göra detta" or "Du _är_ inte _behörig_ att göra detta"